### ISSUE TITLE
Homebridge Beta 1.3.0

### ISSUE THREAD

**Issue by donavanbecker** [bwp91/homebridge-govee#35]
### What issue do you have? Please be as thorough and explicit as possible.
just wanted to let you know about logging that is happening in Homebridge beta 1.3.0 

### Details of your setup.
* Do you use (1) Homebridge (+ config-ui), (2) Homebridge (CLI) or (3) HOOBS? 
Homebridge UI

* Which version of Homebridge/HOOBS do you have?
1.3.0-beta.44

* Which platform do you run Homebridge/HOOBS on (e.g. Raspberry Pi/Windows/HOOBS Box)? Please also mention your version of Node.js/NPM if known.
macOS

* Which version of this plugin (homebridge-govee) do you have? Has the issue started since upgrading from a previous version?
v2.7.1

* Which Govee devices do you have that are causing issues? Please include product models if applicable.
H6163

### Please paste any relevant logs below.

```
[1/13/2021, 1:50:41 PM] Received warning for the plugin 'homebridge-govee' from the characteristic 'Brightness': characteristic was supplied illegal value: number 101 exceeded maximum of 100.
[H6163_3283@@Brightness] characteristic was supplied illegal value: number 101 exceeded maximum of 100.
```

**Comment by bwp91** [comment]
Hey @donavanbecker thanks for this. 
This _surely_ can't happen when you set the brightness - I assume it's when you restart HB it shows then?

**Comment by donavanbecker** [comment]
> Hey @donavanbecker thanks for this. 
> 
> This _surely_ can't happen when you set the brightness - I assume it's when you restart HB it shows then?

Yeah this is on start up. I just set a default to my variable values and then let the refresh of the device info change it.

**Comment by bwp91** [comment]
I think this must be resolved as I introduced the fix where it is _impossible_ for the brightness to be set out of the range of [0, 100]. I'll mark this as complete as the fix was published in v2.7.3 but any further warning messages please do let me know (here or Discord!)
Thanks
